src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -39,7 +39,7 @@ public Manufacturer create(Manufacturer manufacturer) {
     @Override
     public Optional<Manufacturer> get(Long id) {
         String query = "SELECT * FROM manufacturers"
-                + " WHERE id = (?) AND deleted = FALSE";
+                + " WHERE id = (?) AND is_deleted = FALSE";
         try (Connection connection = ConnectionUtil.getConnection();
                 PreparedStatement preparedStatement = connection.prepareStatement(query)) {
```

Comment:
In my opinion, you need to rename "preparedStatement"

Reply:
done